Composer/packages/client/src/pages/knowledge-base/QnAPage.tsx

```diff
@@ -14,7 +14,7 @@ import { navigateTo } from '../../utils/navigation';
 import { TestController } from '../../components/TestController/TestController';
 import { INavTreeItem } from '../../components/NavTree';
 import { Page } from '../../components/Page';
-import { dialogsState, qnaFilesState } from '../../recoilModel/atoms/botState';
```

Comment:
I don't think we should have an index barrel file for the recoilModel.  We found in LUIS how much is slows down build and messes with hot module reload.